CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ### Features
 
+- `[jest-mock]` Update `mockImplementation`, `mockImplementationOnce`, `mockReturnValue`, `mockReturnValueOnce`, `mockResolvedValue`, `mockResolvedValueOnce`, `mockRejectedValue` and `mockRejectedValueOnce` to make their arguments optional ([#14035](https://github.com/facebook/jest/pull/14035))
```

Comment:
Since there is no error at runtime, I think this is just a fix of typings -- "Allow ... methods to be called without argument ..."